src/pages/index.vue

```diff
@@ -1,7 +1,314 @@
+<template>
+  <Page
+    :class="[
+      'w-full',
+      'laptop:px-[32px]',
+      'px-[12px]',
+      'pb-[132px]',
+    ]"
+  >
+    <section
+      :class="[
+        'w-full',
+        'max-w-[960px]',
+      ]"
+    >
+      <h2 class="text-[#3AB7A2] text-[48px] font-proxima font-[600]">{{ $t('home_section_book_title') }}</h2>
```

Comment:
Have book and article in left and right tabs seems better, the current page is too long and articles(WNFT) is too far down below

Reply:
Updated as @joshkiu design

<img width="1026" alt="image" src="https://github.com/likecoin/liker-land/assets/17264716/0e027b4d-336c-4562-ba09-4fd571b9f043">
<img width="602" alt="image" src="https://github.com/likecoin/liker-land/assets/17264716/b6b10c26-406b-4268-b4b6-ce0bc0f0ffd0">
<img width="240" alt="image" src="https://github.com/likecoin/liker-land/assets/17264716/f4f02e23-1155-48d9-a9cb-9f43ca31c3f1">